src/vivarium_cluster_tools/vadmin/oauth_utilities.py

```diff
@@ -180,23 +282,26 @@ def oauth_remove_stash(config: OAuthConfig):
 #############################
 
 def oauth_create_github(config: OAuthConfig):
+    """Creates a new OAuth token on github."""
     user = get_user('github')
     api_endpoint = 'https://api.github.com/authorizations'
 
     payload = {
         'note': 'vadmin init',
-        'scopes': ['user', 'repo']
+        'scopes': ['user', 'repo', 'delete_repo']
     }
+    logger.debug('Generating new OAuth token on github.')
     response = requests.post(api_endpoint, auth=user, data=json.dumps(payload))
     token_config = parse_token_creation_response(response)
     config.update('github', token_config)
 
 
 def oauth_remove_github(config: OAuthConfig):
+    """Creates a new OAuth token on github."""
```

Comment:
I think this should be 'Removes' instead of 'Creates' and do you want a debug msg like in the stash remove method?